app.js

```diff
@@ -23,41 +17,23 @@ app.use(session(config.session));
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
-// Middleware to send user info to layout
-app.use(function (req, res, next) {
-  if (req.session.userInfo) {
-    res.locals.userInfo = req.session.userInfo;
-  }
-
-  res.locals.adminRights = auth.adminRights;
-  next();
-});
-
 // Routes
-var index = require("./routes/index.js");
-app.get('/', index);
-
-app.get('/login', auth.bounce, function (req, res) {
-  auth.setSessionUserInfo(req, function() {
-    res.redirect('/');
-  });
-});
-
-app.get('/logout', function (req, res) {
-  delete req.session.userInfo;
-  delete req.session[auth.userNameSession];
-  res.redirect('/');
-});
-
-var add_user = require("./routes/add_user.js");
-app.post('/addUser', add_user);
-
+app.use('/', require('./routes/users.js'));
 app.use('/', require('./routes/add_location.js'));
```

Comment:
For the sake of a clean slate REST API, could you remove add_location, bandana & refunds as well (both here and in /routes)?
We'll re-add those as soon as they are working.